src/odoc/etc/odoc.css

```diff
@@ -419,28 +419,44 @@ pre code {
   padding: 0.35em 0.5em;
 }
 
-.def-doc {
-  margin-bottom: 10px;
+li:not(:last-child) > .def-doc {
+  margin-bottom: 15px;
 }
 
 /* Spacing between items */
 div.odoc-spec,.odoc-include {
   margin-bottom: 2em;
 }
 
-.spec.type .variant {
+.spec.type .variant p, .spec.type .record p {
+  margin: 5px;
+}
+
+.spec.type .variant, .spec.type .record {
   margin-left: 2ch;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 4px;
 }
-.spec.type .variant p {
-  margin: 0;
-  font-style: italic;
+
+.spec.type .record > code, .spec.type .variant > code {
+  min-width: 40%;
 }
-.spec.type .record {
-  margin-left: 2ch;
+
+.spec.type > ol {
+  margin-top: 0;
+  margin-bottom: 0;
 }
-.spec.type .record p {
-  margin: 0;
-  font-style: italic;
+
+.spec.type .record > .def-doc, .spec.type .variant > .def-doc {
+  min-width:50%;
+  padding-left: 22px;
+  margin-left: 10%;
+  margin-right: 0;
+  border-radius: 3px;
+  flex-grow:1;
```

Comment:
I'm not sure this has any effect. The `min-width` idea already do the "column" layout and this can only grow the empty part of the div into empty space.

Reply:
I would prefer to keep it: If some css theme want to add a background color (I tried that first), it will look better with `flex-grow: 1`. Of course, the css theme can also add this line, but that requires the author to dig into our css file deeper...

Reply:
I see, thanks. I agree it's better if the element grows to the right.